docs/docs/using-pants/advanced-target-selection.mdx

```diff
@@ -185,3 +185,58 @@ For other goals, you can leverage shell piping to partition the input targets in
 ```bash
 pants list :: | awk 'NR % 5 == 0' | xargs pants package
 ```
+
+## Referring to a group of targets
+
+If setting tags on individual targets is not feasible, there are a few other options available to refer to multiple targets.
+
+If you have an operation that you perform often on a certain group of targets, you can use the
+[cli](https://www.pantsbuild.org/2.19/reference/subsystems/cli) subsystem options to create shortcuts. For instance,
+this alias
+
+```toml title="pants.toml"
+publish-libraries = "--filter-target-type=python_distribution --filter-address-regex=\"['^src/libA/,^src/libB/']\" publish src::"
+```
+
+would you let run `pants publish-libraries` to publish all Python distributions declared in the `src/libA` and `src/libB`
+directories.
+
+If you want to be able to run an arbitrary Pants goal on a group of targets, you can create a generic target with no
+specific type named [`target`](https://www.pantsbuild.org/2.19/reference/targets/target):
+
+```python title="BUILD"
+target(name="python-libs", dependencies=["src/python/libraries/libA", "src/python/libraries/libB"])
+```
+
+If declared in the root of your workspace, you can now address the Python libraries by `//:python-libs`:
+
+```bash
+$ pants publish //:python-libs
+````
+
+:::note Tip: create aliases for targets.
```

Comment:
It's easier to link to proper headings, what do you think about making this its own `###` subsection or even `##` section? (Especially given the potential adjustment for `target`)

However, this page seems to be more about CLI usage, so maybe this content would be better suited to a (sub)section within or near https://www.pantsbuild.org/2.19/docs/using-pants/key-concepts/targets-and-build-files#dependencies-field ?

Reply:
Excellent suggestion, I've moved the content over there.